apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -424,6 +428,7 @@ const Ticket = new GQLListSchema('Ticket', {
                 overrideTicketFieldsForResidentUserType(context, resolvedData)
                 await setSectionAndFloorFieldsByDataFromPropertyMap(context, resolvedData)
                 setClientNamePhoneEmailFieldsByDataFromUser(get(context, ['req', 'user']), resolvedData)
+                await setDeadline(resolvedData)
```

Comment:
Please don't mutate function's arguments under the hood. Try to make it more obvious

Reply:
There are several more functions like this